flink/flink-1.14/src/main/java/io/tidb/bigdata/flink/connector/TiDBCatalog.java

```diff
@@ -89,6 +93,14 @@ public TiDBCatalog(String name, String defaultDatabase, Map<String, String> prop
     this.catalogLoadMode =
         CatalogLoadMode.fromString(
             properties.getOrDefault(TIDB_CATALOG_LOAD_MODE, TIDB_CATALOG_LOAD_MODE_DEFAULT));
+
+    // Report telemetry.
+    if (properties
+        .getOrDefault(TIDB_TELEMETRY_ENABLE, TIDB_TELEMETRY_ENABLE_DEFAULT)
+        .equals(TIDB_TELEMETRY_ENABLE_DEFAULT)) {
```

Comment:
why use equals? 
Boolean.getBoolean or Boolean.valueof may be better. the latter can ignore the case

Reply:
@shiyuhang0 Thanks for your review. I will change it to Boolean.getBoolean.